packages/add/README.md

```diff
@@ -23,4 +23,6 @@ add();
 ### CLI (via `webpack-cli`)
 ```bash
 npx webpack-cli add
-```
\ No newline at end of file
+```
+---
+[Back to Packages](https://github.com/webpack/webpack-cli/tree/master/packages)
```

Comment:
For all these navigations: 
Someone who has come to the page by clicking from main README.md can simply click back button of the browser. And jumping on the same page (which is fast) makes more sense.

Reply:
Well web navigation doesn't have a pre-defined flow. There's no home page :) like an app